app/vmalert: fixed path prefixes for system routes (#6435)
VictoriaMetrics/VictoriaMetrics: docs/CHANGELOG.md

* BUGFIX: [Single-node VictoriaMetrics](https://docs.victoriametrics.com/) and `vmselect` in [VictoriaMetrics cluster](https://docs.victoriametrics.com/cluster-victoriametrics/): filter deleted label names and values from [`/api/v1/labels`](https://docs.victoriametrics.com/url-examples/#apiv1labels) and [`/api/v1/label/.../values`](https://docs.victoriametrics.com/url-examples/#apiv1labelvalues) responses when `match[]` filter matches small number of time series. The issue was introduced [v1.81.0](https://docs.victoriametrics.com/changelog_2022/#v1810).
* BUGFIX: [vmalert-tool](https://docs.victoriametrics.com/vmalert-tool/): fix float values template in `input_series`. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6391).
* BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert/): fix misleading error logs in vmalert's stdout when unsupported HTTP path is requested.

> * BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert/): fix system links in vmalert UI for a case, when -http.pathPrefix is set.

Reviewer: ```suggestion
* BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert/): fix system links path in vmalert's UI when `-http.pathPrefix` is set.
```
